dialogflow-cx/shirt-ordering-agent/flows.tf

```diff
@@ -0,0 +1,83 @@
+resource "null_resource" "default_start_flow" {
+  # Use a REST API call (instead of Terraform modules) to modify messages and
+  # routes in the default start flow and since Dialogflow creates this default
+  # start flow automatically
+  provisioner "local-exec" {
```

Comment:
If this works, great; If you have time, I would suggest stuffing this into a script and invoking it there. The security/linting was tricky to get this working but you can follow something like https://github.com/GoogleCloudPlatform/contact-center-ai-samples/blob/53ef6d95a3e1d803de18f5b8ce9312b0a61284cf/dialogflow-cx/vpc-sc-demo/components/reverse_proxy_server/main.tf#L248

Reply:
I'm going to leave this as-is for the moment, I did try a few different approaches before landing on this one. If I moved the command to a script, then I would need to handle the handoff of the Terraform triggers/variables to the bash script as well, which adds more complexity. The nice thing with the current approach is that I can grab attributes from flows, intents, and pages and re-use them in both the provision request and the destroy-time provisioner.